packages/xo-web/src/xo-app/settings/plugins/index.js

```diff
@@ -134,6 +134,9 @@ class Plugin extends Component {
               />
               <span className='text-primary'>{` ${props.name} `}</span>
               <span>{`(v${props.version}) `}</span>
+              {props.description !== '' && (
```

Comment:
`trim` should be used before the test to avoid displaying an empty string.

Reply:
[If we creating a default package.json file with `npm init`, the description will be an empty string.](https://docs.npmjs.com/creating-a-package-json-file#creating-a-default-packagejson-file)

So,  `no description`  will be an empty string and IMHO, we don't need to use `trim`.
What do you think?

Reply:
Package description can be modified by a human and he can let a unwanted spaces.